aemedge/blocks/tabs/tabs.css

```diff
@@ -31,6 +36,24 @@
   width: unset;
 }
 
+.rewards h5{
+  color: #faa21b;
+}
+
+.tabs.wide h1,
+.tabs.wide h2,
+.tabs.wide h3,
+.tabs.wide h4,
+.tabs.wide h5 {
+  text-align: center;
+}
+
+.tabs.wide p {
+  margin-left: 10%;
```

Comment:
The margins aren't needed for mobile, I think you can move these to the 768px width breakpoint.

Reply:
Moved. thanks